### PR TITLE
update srl gnmic request limits

### PR DESCRIPTION
**Pull request by steiler**
implementing #1235

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ on:
   workflow_dispatch:
 
 env:
-  GOVER: 1.18.4
+  GOVER: 1.19.5
```
Comment (hellt) [inline .github/workflows/cicd.yml:14]: Hi @steiler 
can you revert this for now. We will move to 1.20 when 1.21 will come out

File: nodes/srl/srl.go [hunk 2]
```diff
@@ -45,6 +45,9 @@ set / system tls server-profile clab-profile trust-anchor "{{ .TLSAnchor }}"
 set / system tls server-profile clab-profile authenticate-client false
 {{- end }}
 set / system gnmi-server admin-state enable network-instance mgmt admin-state enable tls-profile clab-profile
+set / system gnmi-server rate-limit 65000
+set / system gnmi-server commit-save false
```
Comment (hellt) [inline nodes/srl/srl.go:49]: let's remove this, since this is a default

File: .github/workflows/cicd.yml (hunk 1 above)
Comment (steiler) [reply .github/workflows/cicd.yml:14]: then our tests fail ... take a look at this:
https://github.com/srl-labs/containerlab/actions/runs/4085203521/jobs/7042866028
happened to me also in the other PR

Comment (hellt) [reply .github/workflows/cicd.yml:14]: then we have to pin staticcheck to `v0.4.0` in the install command to keep 1.18 requirement

File: nodes/srl/srl.go (hunk 2 above)
Comment (steiler) [reply nodes/srl/srl.go]: done

**Pull request merged**